.github/docker/centreon-web/alma9/entrypoint/container.sh

```
@@ -7,5 +7,9 @@ set -x
 # ls is required to ensure that the scripts are properly sorted by name.
 BASEDIR="/usr/share/centreon/container.d"
 for file in `ls $BASEDIR` ; do
-  . "$BASEDIR/$file"
+  if [[ "$file" == *"_background"* ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)